android/build.sh

```diff
@@ -108,8 +108,18 @@ function archive_artifacts() {
   fi
 }
 
-# Override JAVA_HOME to OpenJDK 11
-set_java_home
+
+# For CI compatibility of building Keyman for Android 18.0 with OpenJDK 11,
+# this overrides JAVA_HOME for the builder script to use OpenJDK 21.
+android_set_java_home() {
+  if [[ ! -z ${JAVA_HOME_21+x} ]]; then
+    builder_echo "Setting JAVA_HOME to JAVA_HOME_21 (${JAVA_HOME_21})"
+    export JAVA_HOME="${JAVA_HOME_21}"
+  fi
+}
+
+# Override JAVA_HOME
+android_set_java_home
```

Comment:
I wonder if we should move this to the TC script wrappers?

Reply:
I think I prefer a general location since it's used in local dev builds